testing/mbuffer/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Katie Holly <holly@fuslvz.ws>
+# Maintainer: Katie Holly <holly@fuslvz.ws>
+pkgname=mbuffer
+pkgver=20171011
+pkgrel=0
+pkgdesc="mbuffer buffers I/O operations and displays the throughput rate"
+url="http://www.maier-komor.de/mbuffer.html"
+arch="all"
+license="GPL-3.0"
```

Comment:
We are moving towards [SPDX](https://spdx.org/licenses/) so it should be `license="GPL-3.0-only"`.